applovin_max/example/lib/main.dart

```diff
@@ -82,7 +85,8 @@ class _MyAppState extends State<MyApp> {
         _interstitialRetryAttempt = _interstitialRetryAttempt + 1;
 
         int retryDelay = pow(2, min(6, _interstitialRetryAttempt)).toInt();
-        logStatus('Interstitial ad failed to load with code ${error.code} - retrying in ${retryDelay}s');
+        logStatus(
+            'Interstitial ad failed to load with code ${error.code} - retrying in ${retryDelay}s');
```

Comment:
Can we fix the formatting? It's not a pretty way to call into `logStatus` function. Maybe you can create a formatting similar to applovin_code_styles.xml in Android-Workspace

Reply:
Just realized there is an option to increase the line length in Android Studio.